mod_reforged/hooks/entity/tactical/human.nut

```diff
@@ -4,5 +4,6 @@
 	{
 		onInit();
 		this.m.BaseProperties.Reach = ::Reforged.Reach.Default.Human;
+		this.m.Skills.add(::new("scripts/skills/actives/recover_skill"));
```

Comment:
I agree. I can change it to `getSkills()`. I was just going with the vanilla method of using `this.m.Skills` in the `onInit` function. I imagine it is faster by a very small amount because of reducing one function call. But yeah we should use `getSkills()`.

Reply:
I decided to stick with `this.m.Skills` for now as that is the standard vanilla style in all `onInit` and similar functions. If we want to do a refactor of this we should do it for all skills or leave this as it is for entity files.